Doc/library/profile.rst

```diff
@@ -141,6 +149,9 @@ the output by. This only applies when ``-o`` is not supplied.
    .. versionadded:: 3.8
       Added the ``-m`` option to :mod:`profile`.
 
+Note that if the process is terminated by ``SIGTERM`` during profiling, no
```

Comment:
How about SIGKILL? ;)

Reply:
Good point, I changed the description a bit as there might be other cases when the profiling won't work.